spec/genre_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rspec'
```

Comment:
- [ ] Just as i suggested above, you need to remove the `require 'rspec'` statement as there is no `rspec gem` in the Gemfile. :wave: :+1: